tests/PHPStan/Rules/Comparison/data/check-type-function-call.php

```diff
@@ -32,7 +32,7 @@ public function doFoo(
 
 		}
 		$className = 'Foo';
-		if (is_a($className, \Throwable::class, true)) { // should be fine
+		if (is_a($className, \Throwable::class, true)) { // always false
```

Comment:
is no longer considered "fine" because `Foo` is not a subclass of `Throwable`

Reply:
But also please test `Foo::class` (which is different thanks to namespace resolution), the class isn't final so a subclass might implement Throwable.

Reply:
hm I guess you mean I should test `class-string<Foo>` as a parameter not `Foo::class`.. at least thats what I did for now.. see added tests at the end of this file